keylime/tenant.py

```diff
@@ -983,7 +989,7 @@ def do_quote(self):
             raise UserError(f"Status command response: {response.status_code} Unexpected response from Cloud Agent.")
 
         if "results" not in response_body:
-            raise UserError(f"Error: unexpected http response body from Cloud Agent: {str(response.status)}")
+            raise UserError(f"Error: unexpected http response body from Cloud Agent: {str(response_body)}")
```

Comment:
I don't know if we actually want to log the entire body. I think the status code is fine.

Reply:
da2288aded96fcf8ec3a9b3ece9dce994e173e55